tox.ini

```diff
@@ -3,12 +3,12 @@ envlist =
     py{3.6,3.7,3.8}-django-{22,30,31,master}
 
 [testenv]
-commands = ./test.sh {posargs}
+allowlist_externals = poetry
```

Comment:
Instead of using an external poetry, you could have tox manage installation of poetry itself. If you want to keep using tox.